### DISCUSSION TITLE
Where did the overrides hotkeys go?

### DISCUSSION THREAD

**Discussion by Smoghog** [winder/Universal-G-Code-Sender#1574] [Help]
Sorry to have another issue so soon after my last, but... I updated to the latest nightly so I could use the shortcut to jump to console entry for my macro pad, but now the hotkeys for various overrides are no longer available for customization, so I can't use my Feed Rate Override Increase and Decrease buttons during the carve. 

Are these gone for good? Or is this a potential bug?

**Comment by breiler** [comment]
This is probably a bug, I'll try to investigate this later this weekend

**Reply by Smoghog** [reply]
Did you have any luck figuring this issue out?
